src/sql/src/plan/statement/ddl.rs

```diff
@@ -2181,9 +2184,12 @@ pub fn plan_create_sink(
     scx.require_unsafe_mode("CREATE SINK")?;
     let compute_instance = match &stmt.in_cluster {
         None => scx.resolve_compute_instance(None)?.id(),
-        Some(in_cluster) => in_cluster.0,
+        Some(in_cluster) => in_cluster.id,
     };
-    stmt.in_cluster = Some(ResolvedClusterName(compute_instance));
+    stmt.in_cluster = Some(ResolvedClusterName {
+        id: compute_instance,
+        print_name: None,
```

Comment:
Why does `print_name` always get set to `None`? Can't you get the name from either `in_cluster.print_name` or `scx.resolve_compute_instance(None)?.name()`?

Reply:
Ah because the field is a bit of a hack (akin to the existing `print_id` bool) to allow the `NameSimplifier` to indicate which IDs can be simplified. I left a comment to that effect.